godot_project/editor/input_handlers/molecular_structures/selection_input_handler.gd

```diff
@@ -116,6 +116,14 @@ func forward_input(in_input_event: InputEvent, in_camera: Camera3D, in_context:
 		var hovering_bond_id: int = -1
 		var hovering_spring_id: int = -1
 		var hover_position: Vector3 = Vector3(INF, INF, INF)
+		var rendering: Rendering = _workspace_context.get_rendering()
+		var hovering_an_autopose_control: bool = (
+				rendering.atom_autopose_preview_is_visible() and
+				rendering.atom_autopose_get_hovered_candidate_or_null() != null
+		)
+		if hovering_an_autopose_control:
+			get_workspace_context().set_hovered_structure_context(null, -1, -1, -1)
+			return false
```

Comment:
@kubecz3k any idea of why despite this is being executed properly the application still gives priority to hovering selection? :(

Reply:
Actually what you expect for it to do when you are pushing null there? To 'unhover' the structure context? I think we should introduce new function in order to do that